app/controllers/communitarian/page_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Communitarian
+  class PageController < ::ApplicationController
+    requires_plugin Communitarian
+
+    def index
+      render json: { title: "Hello world!" }
```

Comment:
do we really need this action?)

Reply:
With the current implementation, most likely not needed. But what if we move the logic for getting categories here from the home route. Does this make sense and benefits?

Reply:
Removed this action